research/picai/data_utils.py

```diff
@@ -169,28 +178,29 @@ def get_img_and_seg_paths(
             torch tensor that give the class proportions.
     """
 
-    # load datasheets
-    file_name = f"PI-CAI_train-fold-{fold_id}.json" if train else f"PI-CAI_val-fold-{fold_id}.json"
+    # Make sure at least one sequence will be present
+    assert not (include_t2w is False and include_adc is False and include_hbv is False)
```

Comment:
Super minor, but I think you can do this slightly more cleanly with 
`assert any([include_t2w, include_adc, include_hbv])`